src/app.js

```diff
@@ -22,9 +25,22 @@ app.use(bodyParser.json({ type: 'application/vnd.api+json' }));
 app.use(methodOverride('X-HTTP-Method-Override'));
 app.use(passport.initialize());
 
-app.use('/users', usersRouter);
+const userPermissionCheck = [
+  jwtValidator,
+  permissionsValidator(permissions.USER)
+];
+const adminPermissionCheck = [
+  jwtValidator,
+  permissionsValidator(permissions.ADMIN)
+];
+
+// public routes:
 app.use('/authentication', authenticationRouter);
-app.use('/geolocation', geolocationRouter);
-app.use('/events', eventsRouter);
+// user routes:
+app.use('/myprofile', ...userPermissionCheck, userProfileRouter);
```

Comment:
First... How do you specify permission for each route within the users/, for example I want
`/users/create` to be public, but
`/users/load` to be admin only

Reply:
Second... You can simplify this.

Make your own route constructor, that way you can incapsulate this kind of code.